extensions/amp-autocomplete/validator-amp-autocomplete.protoascii

```diff
@@ -41,6 +41,7 @@ tags: {  # <amp-autocomplete>
       if_value_regex: "custom"
       also_requires_attr: "filter-expr"
     }
+    mandatory: true
```

Comment:
This appears to add a new constraint that wasn't there before. Previously "filter" was only required if "suggest-first" was present. Now it's already required. This will probably break existing documents, which is undesirable. Have you considered what the impact of that will be?

Reply:
Today `amp-autocomplete` will break during the `buildCallback` if `filter` isn't present, so it is both desirable and somewhat expected for publishers to be using this attribute anyway. I will look into usage to see if this is true in practice.

Reply:
That probably indicates this is OK, but a broken autocomplete is a different impact to publisher than an invalid AMP page. We should do further analysis.

Reply:
FYI, if you need help with that, let me know.

Reply:
Looked into this and there is 100% overlap between live usage of `amp-autocomplete` and `amp-autocomplete[filter]`